src/SFML/Graphics/Font.cpp

```diff
@@ -583,11 +583,14 @@ Glyph Font::loadGlyph(Uint32 codePoint, unsigned int characterSize, bool bold, f
         // pollute them with pixels from neighbors
         const unsigned int padding = 1;
 
+        width += 2 * padding;
+        height += 2 * padding;
```

Comment:
The block of memory we work with now includes the border padding (which it didn't before). Instead of initializing the texture to the padding colour when we create it (which might waste time when the texture gets really big), we do it every time a glyph would actually need the padding around it and upload it then.